src/recipes/ceviche-with-shrimp.md

```diff
@@ -0,0 +1,42 @@
+---
+tags: [side dish]
+title: "Ceviche With Shrimp"
+---
+
+# Ceviche With Shrimp
+
+Ceviche is one of my favorite dishes that I enjoyed when I started my no red meat diet. I first tried Ceviche when my bestfriend brought it to work for Lunch. I loved it so much that I learned how to make it on my own.
+
+## Ingredients
+
+1 bag of cooked Shrimp
```

Comment:
please make these and other lists bullet points to format well: 

- ingredient 1
- ingredient 2